apps/backend/src/lambdas/dailyfilecheck.ts

```diff
@@ -20,33 +19,54 @@ export const handler = async (event: unknown, context?: Context) => {
   appLogger.log({ event });
   appLogger.log({ context });
 
-  const date = format(new Date(), 'yyyy-MM-dd');
+  const date = new Date();
 
-  const dailyAlert: DailyAlertRO = await notificationService.dailyUploadAlert(
-    date
-  );
+  // Retrieve statuses from the past 7 days
+  const allRecentStatuses =
+    await notificationService.retrieveRecentDailyStatuses(
+      subDays(date, 7),
```

Comment:
do we care about business days? (or does 7 normal days cover as you are only trying to look back 5?)

Reply:
I should remove this magic number. But essentially, yeah the buffer is pretty arbitrary, it's as if to say "enough days have passed and enough eyes should have seen this that we're going to continue reconciling if more recent files have been uploaded"

Reply:
Updated